setup.py

```diff
@@ -10,7 +10,7 @@
     packages=find_packages(exclude=['test', 'test.*']),
     install_requires=[
         "nameko>=2.7.0",
```

Comment:
This needs to be >= 2.11.0. Nameko dropped support for Kombu 3 when it started supporting Kombu 4. I don't think it's necessary (or possible without feature-sniffing) for this library to support older Nameko (and Kombu 3).